go-controller/pkg/ovn/egressip_local_test.go

```diff
@@ -418,8 +418,8 @@ var _ = Describe("Local gateway mode EgressIP Operations with", func() {
 
 				fakeOvn.fakeExec.AddFakeCmd(
 					&ovntest.ExpectedCmd{
-						Cmd:    fmt.Sprintf("ovn-nbctl --timeout=15 --data=bare --format=table --no-heading --columns=options find logical_router name=GR_%s options:lb_force_snat_ip!=-", node2.name),
-						Output: fmt.Sprintf("lb_force_snat_ip=%s", nodeLogicalRouterIPv6),
+						Cmd:    fmt.Sprintf("ovn-nbctl --timeout=15 --format=table --no-heading --columns=options find logical_router name=GR_%s options:lb_force_snat_ip!=-", node2.name),
```

Comment:
I think these changes should be a part of commit `adfc86f54a7c766a3fd4c4ca9894f817fa2626eb`, seeing as how they're linked to that change.

Reply:
hm? no they're not

(just tested again: `make test` passes for each commit)